Makefile.dep

```diff
@@ -607,6 +608,13 @@ ifneq (,$(filter random,$(USEMODULE)))
     USEMODULE += prng_tinymt32
   endif
 
+  ifneq (,$(filter prng_fortuna,$(USEMODULE)))
```

Comment:
btw. with the above applied, this would go into `sys/Makefile.dep` right?

Reply:
I think so. Not sure, because I basically duplicated `prng_tinymt32`.

Reply:
see comment above, leave as is for now and we adapt all sys-modules in a separate cleanup-PR.